tests/spread/plugins/craft-parts/build-and-run-hello/uv-hello/src/pyproject.toml

```
@@ -0,0 +1,16 @@
+[project]
+name = "example"
```

It may be worth setting a name here to something other than `example`
```suggestion
name = "hello"
```